.github/workflows/github_actions_aws_win_py32.yml

```diff
@@ -5,16 +5,18 @@ on:
     branches:
       - master
 jobs:
-  run-system-tests-python-32bit:
-    runs-on:
-      - self-hosted
-      - windows
-      - x64
-      - rdss-nidaqmxbot-win-10-py32
-    timeout-minutes: 80
-    steps:
-      - uses: actions/checkout@v3
-      - name: Install dependencies
-        run: poetry install
-      - name: Run system tests
-        run: poetry run tox
\ No newline at end of file
+  do-the-job1:    
+    run-system-tests-python-32bit:
```

Comment:
You can only define one level of job id.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_id

Reply:
Removed the second level job id.